keyboards/handwired/wulkan/config.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Wulkan
+#define PRODUCT         Handwired48Keys
+#define DESCRIPTION     A compact ortholinear handwired keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 4
+#define MATRIX_COLS 12
+
+#define MATRIX_ROW_PINS { B8, A0, A1, A2 }
+#define MATRIX_COL_PINS { B13, B14, B15, B9, B7, B6, B5, B4, B3, B2, B1, B0 }
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+#define UNICODE_KEY_LNX  LCTL(LSFT(KC_U))
```

Comment:
This isn't actually needed here, as this is the default value.  You'd only need this if you were using something else.  And even then, it would be better to place that in the keymap's config.h file. 

```suggestion
```

Reply:
Yes, don't know why I left that one in there..